pkg/apis/resource/types.go

```diff
@@ -219,6 +221,40 @@ type BasicDevice struct {
 	//
 	// +optional
 	Capacity map[QualifiedName]DeviceCapacity
+
+	// UsageRestrictedToNode indicates if the usage of an allocation involving this device
+	// has to be limited to exactly the node that was chosen when allocating the claim.
+	//
+	// +optional
+	UsageRestrictedToNode *bool
```

Comment:
```suggestion
	// +featureGate=DRADeviceBindingConditions
	UsageRestrictedToNode *bool
```

Please also add the "This is an alpha field and requires ..." paragraph as in other such fields, see https://github.com/kubernetes/kubernetes/pull/130447.

Reply:
What about `BoundToNode` as field name?

Reply:
The device is not bound to a node yet. I like `UsageRestrictedToNode` better.